src/components/data/FileBrowser.js

```diff
@@ -344,7 +345,7 @@ const BucketBrowser = ({
   const [deletingSelectedObjects, setDeletingSelectedObjects] = useState(false);
   const [busy, setBusy] = useState(false);
 
-  const editWorkspaceError = Utils.editWorkspaceError(workspace);
+  const editWorkspaceError = WorkspaceUtils.editWorkspaceError(workspace);
   const canEditWorkspace = !editWorkspaceError;
```

Comment:
I see many calls where `editWorkspaceError()` is negated to produce `canEditWorkspace`. Also defining a `canEditWorkspace()` function could make this code clearer.

Reply:
I see that. Since this is a move/rename PR, I would not make that change here, but I think that would help a lot with readability.